src/components/MultiHopTrade/hooks/useIsTradingActive.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react'
+import { getIsTradingActiveApi } from 'state/apis/swapper/getIsTradingActiveApi'
+import { selectActiveSwapperName } from 'state/slices/tradeQuoteSlice/selectors'
+import { useAppDispatch, useAppSelector } from 'state/store'
+import { selectBuyAsset, selectSellAsset } from 'state/zustand/swapperStore/selectors'
+import { useSwapperStore } from 'state/zustand/swapperStore/useSwapperStore'
+
+export const useIsTradingActive = () => {
+  const [isTradingActiveOnSellPool, setIsTradingActiveOnSellPool] = useState(false)
+  const [isTradingActiveOnBuyPool, setIsTradingActiveOnBuyPool] = useState(false)
+
+  const dispatch = useAppDispatch()
+
+  const buyAsset = useSwapperStore(selectBuyAsset)
+  const sellAsset = useSwapperStore(selectSellAsset)
+  const sellAssetId = sellAsset?.assetId
+  const buyAssetId = buyAsset?.assetId
```

Comment:
in the new implementations these should always be defined
```suggestion
  const sellAssetId = sellAsset.assetId
  const buyAssetId = buyAsset.assetId
```

Reply:
not 100% sure what im saying is correct

Reply:
You're correct! Fixed in https://github.com/shapeshift/web/pull/4889/commits/3c2dfb0db4a92e0f1a79c3401a5342d15ab69a42.